Packs/SekoiaXDR/Integrations/SekoiaXDR/README.md

```diff
@@ -0,0 +1,893 @@
+| SekoiaXDR.Events.Status.expiration_date | unknown | The date when the event expires. | 
+| SekoiaXDR.Events.Status.created_at | unknown | The time when the event was created. | 
+| SekoiaXDR.Events.Status.status | unknown | The current status of the event. | 
+| SekoiaXDR.Events.Status.view_uuid | unknown | The view associated with the event. | 
+| SekoiaXDR.Events.Status.canceled_at | unknown | The time when the event was canceled. | 
+| SekoiaXDR.Events.Status.only_eternal | unknown | Whether only eternal events are associated with the event. | 
+| SekoiaXDR.Events.Status.results_ttl | unknown | The time-to-live for the event results. | 
+| SekoiaXDR.Events.Status.canceled_by | unknown | The user who canceled the event. | 
+| SekoiaXDR.Events.Status.term_lang | unknown | The language associated with the term for the event. | 
+| SekoiaXDR.Events.Status.ended_at | unknown | The time when the event ended. | 
+| SekoiaXDR.Events.Status.earliest_time | unknown | The earliest time associated with the event. | 
+| SekoiaXDR.Events.Status.max_last_events | unknown | The maximum number of events to retrieve. | 
+| SekoiaXDR.Events.Status.canceled_by_type | unknown | The type of user who canceled the event. | 
+| SekoiaXDR.Events.Status.total | unknown | The total number of events associated with the event. | 
+| SekoiaXDR.Events.Status.created_by_type | unknown | The type of user who created the event. | 
+| SekoiaXDR.Events.Status.community_uuids[0] | unknown | The community associated with the event. | 
+| SekoiaXDR.Events.Status.filters.field | unknown | The field used for filtering events. | 
+| SekoiaXDR.Events.Status.filters.field | unknown | The field used in the filter. | 
+| SekoiaXDR.Events.Status.filters.value | unknown | The value of the filter. | 
+| SekoiaXDR.Events.Status.filters.operator | unknown | The operator used in the filter. | 
+| SekoiaXDR.Events.Status.filters.excluded | unknown | A boolean indicating whether the filter is excluded or not. | 
+| SekoiaXDR.Events.Status.filters.disabled | unknown | A boolean indicating whether the filter is disabled or not. | 
+
+### sekoia-xdr-events-results-query
+
+***
+Command to retrieve the events from the search job "sekoia-xdr-execute-events-query" previously done on Sekoia XDR.
+
+#### Base Command
+
+`sekoia-xdr-events-results-query`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| uuid | UUID from response of the query executed previously with the "sekoia-xdr-query-events" command. | Required | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| SekoiaXDR.Events.Results | unknown | The outputs are different for each event, they will be output inside SekoiaXDR.Events.Results. | 
+
+### sekoia-xdr-search-events
+
+***
+Command to search and retrieve the events from an alert. This is a combination of 3 commands: jobquery-events, jobquery-events-status and jobquery-retrieve events.
```

Comment:
Please update the names of the commands so it will be clear to the user which are the commands.
Is jobquery-events-status = sekoia-xdr-events-status-query?
Update it in the YML file as well.

Reply:
Thank for this catch